postal/pyparser.c

```diff
@@ -59,7 +59,7 @@ static PyObject *py_parse_address(PyObject *self, PyObject *args, PyObject *keyw
     char *country = NULL;
 
     if (arg_country != Py_None) {
-        country = PyObject_to_string(arg_language);
+        country = PyObject_to_string(arg_country);
         if (country == NULL) {
             goto exit_free_language;
```

Comment:
should we not make it as goto exit_free_country

Reply:
no, I don't think so. We jump when `country` couldn't be allocated, so then we cannot free it. We only need to free what was allocated _before_, that is `language` and `input`.